src/bugjira/bugjira.py

```diff
@@ -0,0 +1,66 @@
+from bugjira.bugzilla import Bugzilla
```

Comment:
in a different file this import is BzBugzilla - just  noting where I had to dig to understand